webserver.go

```
@@ -1986,6 +2101,705 @@ func handleObservabilitySummary(c *gin.Context) {
+	return records, rows.Err()
+}
+
+func renameServerFactsTx(tx *sql.Tx, oldName, newName string) error {
+	if strings.TrimSpace(oldName) == "" || strings.TrimSpace(newName) == "" || oldName == newName {
+		return nil
+	}
+	_, err := tx.Exec("UPDATE server_facts SET server_name = ? WHERE server_name = ?", newName, oldName)
+	return err
+}
+
+func deleteServerFacts(name string) error {
```

golangci-lint (2.11.4)

[error] func deleteServerFacts is unused

(unused)